src/DFTK.jl

```diff
@@ -89,6 +89,7 @@ export spin_density
 export ρ_from_total_and_spin
 include("densities.jl")
 include("interpolation.jl")
+export transfer_blochwave
```

Comment:
If we make this part of the interface than we should consider doing the same for the density for consistency. Are we sure this is now settled enough to commit to this?

Reply:
No, I did it to make it easier for me to use it, but we can remove it if you prefer !

Reply:
I don't feel we are settled enough on this stuff to make it part of the public interface.

Reply:
Ok, I remove it then ! :)